### ISSUE TITLE
v1.1.1 throwing error: Class "WPGraphQL\ContentBlocks\Utilities\WPGraphQLHelpers" not found

### ISSUE THREAD

**Issue by CherryGot** [wpengine/wp-graphql-content-blocks#132]
This is the error message in my debug logs when I try consuming one of the blocks using graphql. 
```
[18-Jul-2023 06:28:02 UTC] PHP Fatal error:  Uncaught Error: Class "WPGraphQL\ContentBlocks\Utilities\WPGraphQLHelpers" not found in /var/www/wordpress/web/app/mu-plugins/wp-graphql-content-blocks/includes/Registry/Registry.php:173
```

The problem:
The `/includes/utilities` does not follow the PSR-4 standard and there the composer dump-autoload may have not produced a rule to resolve the `Utilities` in the namespaces. 

Solution: 
As soon as I changed the utilities directory name to Utilities, made a autload dump, the error disappeared. I think you guys should look into that, just to you know check if there are any side-effects of this change. I'm surprised this issue has not been mentioned in the active set of issues.

**Comment by CherryGot** [comment]
Here's how I'm fixing it at the moment.

```"post-package-install": [
      "cd wordpress/web/app/mu-plugins/wp-graphql-content-blocks/; mv ./includes/utilities ./includes/Utilities; composer dump-autoload -o"
    ]```

**Comment by CherryGot** [comment]
I used composer to install the plugin because we are using it to manager all our plugins, like so, 
```composer require wpengine/wp-graphql-content-blocks```

Although, the documentation did not mention it, the composer require worked just fine. But yeah, maybe it's one of those things that it was never tested for. 

Anyways, thanks for the considering the change, it'd be huge help.

**Comment by theodesp** [comment]
Hey @CherryGot we don't register the plugin in composer registry for now. Maybe thats the issue.

**Comment by CherryGot** [comment]
Yeah, agree. We are not using the plugin from composer registry but with repositories field of composer.json, like so

```JSON
"repositories": [
    {
      "type": "github",
      "url": "https://github.com/wpengine/wp-graphql-content-blocks.git"
    }
  ]
  ```

**Comment by justlevine** [comment]
@CherryGot we're installing `wp-graphql-content-blocks` as a `vcs` composer repository,  and other than the silly backend warning (since the vendor files are not included locally) I'm not seeing any errors regarding WPGraphQL helpers.

```json
"repositories": [
  {
    "type": "vcs",
    "url": "https://github.com/wpengine/wp-graphql-content-blocks.git"
  },
 ]
 ```
 
Could be that its `type:github` specific, or could be the fatal warning is only triggered sometimes (e.g. on specific blocks).

**Comment by CherryGot** [comment]
The type:github can be a reason. But I guess my focus will be on "triggered sometimes". Yes, this happened only when I was trying to access the attributes of a paragraph block. Other times, it didn't throw any errors. Sorry, I should've mentioned it in the original comment.

Like
```
... on CoreParagraph {
    attributes {
        content
    }
}
```

**Comment by blakewilson** [comment]
Wanted to follow up here that 1.1.3 has been released with the `Utilities` folder rename:

https://github.com/wpengine/wp-graphql-content-blocks/releases/tag/v1.1.3
